src/pages-helpers/curriculum/xlsx/helper.test.ts

```diff
@@ -192,3 +202,294 @@ describe("addOrUpdateSheet", () => {
+      phaseSlug: "secondary",
+      ks4OptionSlug: "gcse",
+      keyStageSlug: undefined,
+      tierSlug: undefined,
+      childSubjectSlug: undefined,
+    };
+    const output = generateYearTitle(formattedData, year, slugs);
+    expect(output).toEqual("Year 11 Physical education, GCSE");
+  });
+
+  test("Year 11 Chemistry, Foundation", () => {
+    const formattedData = {
+      yearData: {
+        "11": createYearData({
+          units: [
+            createUnit({
+              year: "11",
+              subject_slug: "chemistry",
+              examboard: "AQA",
+              tier: "Foundation",
+            }),
+          ],
+        }),
+      },
+      threadOptions: [],
+      tiers: ["foundation"],
+      yearOptions: ["11"],
+    };
+    const year = "11";
+    const slugs = {
+      subjectSlug: "science",
+      phaseSlug: "secondary",
+      ks4OptionSlug: "gcse",
+      keyStageSlug: undefined,
+      tierSlug: "foundation",
+      childSubjectSlug: "chemistry",
+    };
+    const output = generateYearTitle(formattedData, year, slugs);
+    expect(output).toEqual("Year 11 Chemistry, Foundation");
+  });
+
+  test("Year 10 Combined science, Higher", () => {
+    const formattedData = {
+      yearData: {
+        "11": createYearData({
+          units: [
+            createUnit({
+              year: "11",
+              subject_slug: "chemistry",
+              examboard: "AQA",
+              tier: "Foundation",
+            }),
+          ],
+        }),
+      },
+      threadOptions: [],
+      tiers: ["foundation"],
+      yearOptions: ["11"],
+    };
+    const year = "11";
+    const slugs = {
+      subjectSlug: "science",
+      phaseSlug: "secondary",
+      ks4OptionSlug: "gcse",
+      keyStageSlug: undefined,
+      tierSlug: "higher",
+      childSubjectSlug: "chemistry",
+    };
+    const output = generateYearTitle(formattedData, year, slugs);
+    expect(output).toEqual("Year 11 Chemistry, Higher");
+  });
+
+  // When a year has a subject title override (i.e. KS4 GCSE Computing = Computer science), update the subject name accordingly in the sheet title.
+  test("Year 10 Computer science, GCSE", () => {
+    const formattedData = {
+      yearData: {
+        "10": createYearData({
+          units: [
+            createUnit({
+              year: "10",
+              subject_slug: "computing",
+              examboard: "AQA",
+            }),
+          ],
+        }),
+      },
+      threadOptions: [],
+      tiers: ["foundation"],
+      yearOptions: ["10"],
+    };
+    const year = "10";
+    const slugs = {
+      subjectSlug: "computing",
+      phaseSlug: "secondary",
+      ks4OptionSlug: "aqa",
+    };
+    const output = generateYearTitle(formattedData, year, slugs);
+    expect(output).toEqual("Year 10 Computer Science, GCSE");
+  });
+
+  // Year groups that do not have these modifiers (i.e. Years 1-9) should have default titles
+  test("Year 9 Physical education", () => {
```

Comment:
This test is passing, but based on the actual output it shouldn't be. Secondary documents which have KS4 programme factors (i.e. tier or pathway) are still outputting these modifiers in KS3 sheet titles:

<img width="1419" height="885" alt="Screenshot 2025-08-13 at 10 00 09" src="https://github.com/user-attachments/assets/d96d5c3e-d49a-4ed4-b575-654c0ab44c3e" />
<img width="1407" height="881" alt="Screenshot 2025-08-13 at 10 00 50" src="https://github.com/user-attachments/assets/f8abc261-1845-4d16-bbaa-d7616bdadc23" />